### PR TITLE
Run jaxrs-2.0 CDI and Client tests with the jaxrs-2.1 feature

### PR DESCRIPTION
**Pull request by WhiteCat22**
PR for #3847 to run the com.ibm.ws.jaxrs.2.0.client_fat and com.ibm.ws.jaxrs.2.0.cdi.1.2_fat tests with the jaxrs-2.1 feature.

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
build shouldn't actually run these tests since the tests still exist in Commercial Liberty

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
Latest build is good, the 2 failures are known issues unrelated to my changes. Just awaiting approval to merge.

File: dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/bnd.bnd [hunk 1]
```diff
@@ -24,6 +24,12 @@ src: \
 
 fat.project: true
 
+# These features get added programatically
+tested.features: \
+  jaxrs-2.1,\
+  cdi-2.0,\
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/bnd.bnd]: if we are repeating these tests for JAX-RS 2.1 and CDI-2.0 also, lets take the feature version numbers out of the bucket name and just call it `com.ibm.ws.jaxrs.cdi_fat`

File: dev/com.ibm.ws.jaxrs.2.0.client_fat/bnd.bnd [hunk 2]
```diff
@@ -39,6 +39,13 @@ src: \
 
 fat.project: true
 
+# These features get added programatically
+tested.features: \
+  jaxrsclient-2.1,\
+  jaxrs-2.1,\
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxrs.2.0.client_fat/bnd.bnd]: again, if we are repeating the tests with different JAX-RS levels, should we take the version number out of the bucket name?

File: dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/ServerConfigurationFactory.java [hunk 3]
```diff
@@ -165,8 +165,8 @@ public static void main(String[] args) throws Exception {
         HttpEndpoint httpEndpoint = new HttpEndpoint();
         httpEndpoint.setId("defaultHttpEndpoint");
         httpEndpoint.setHost("*");
-        httpEndpoint.setHttpPort(Integer.valueOf(9080));
-        httpEndpoint.setHttpsPort(Integer.valueOf(9443));
+        httpEndpoint.setHttpPort(Integer.toString(9080));
```
Comment (aguibert) [inline dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/ServerConfigurationFactory.java]: could just put the literal string "9080" in here

**Pull request merged**